src/test/java/uk/nhs/tis/trainee/notifications/service/EmailServiceIntegrationTest.java

```diff
@@ -213,8 +223,18 @@ void shouldGreetExistingUsersConsistentlyWhenNameProvided(NotificationType notif
     when(userAccountService.getUserDetailsById(USER_ID)).thenReturn(
         new UserDetails(true, RECIPIENT, null, "Gilliam", "Anthony", null));
 
+    Instant timestamp = Instant.now();
+    Map<String, Object> variables = Map.of("formRef", "form ref",
+        "state", "some state",
+        "timestamp", timestamp,
+        "formName", "form name",
+        "discussions", Map.of("tpdName", "TPD name"),
+        "programmeMembership", Map.of("startDate", "2024-01-01",
+            "name", "PM name", "wte", 1.0, "managingDeanery", "MD name"),
+        "personalDetails", Map.of("gmcNumber", "1234567"),
+        "change", Map.of("startDate", "2024-01-01", "wte", 0.5, "cctDate", "2024-01-01"));
     service.sendMessageToExistingUser(PERSON_ID, notificationType, templateVersion,
-        Map.of("familyName", "Maillig"), null);
+        Map.of("familyName", "Maillig", "var", variables), null);
```

Comment:
🤢 . But...why? Because otherwise I can't seem to get it to work. If I include a simple` "discussions", Map.of("tpdName", "TPD name"` in the template map it is ignored, and if I nest it in a `vars` map, then suddenly all the elvis null safe checks in thymeleaf seem to stop working, requiring all these other fields to be passed as well.